content/doc/tutorials/tutorial-for-installing-jenkins-on-IBM-Cloud.adoc

```diff
@@ -5,202 +5,225 @@ section: doc
 ---
 
 :toc:
-:toclevels: 3
+:toclevels: 4
 :imagesdir: ../../book/resources/
 
-You should have an IBM Cloud account, otherwise you can http://cloud.ibm.com/registration[register here].
+You should have an IBM Cloud account, otherwise you can link:http://cloud.ibm.com/registration[register here].
 At the end of the tutorial you will have a cluster with Jenkins up and running.
```

Comment:
We could rephrase this slightly, so it reads more as an introduction.
```suggestion
This tutorial demonstrates how to deploy Jenkins on IBM Cloud. Before deploying, you should have an IBM Cloud account, otherwise you can link:http://cloud.ibm.com/registration[register here].
At the end of the tutorial, you will have a cluster with Jenkins up and running.
```